docker-compose.yml

```diff
@@ -160,3 +160,21 @@ services:
     depends_on:
       - handshake
       - handshake-api
+
+  synckeys:
+    build:
+      context: docker/synckeys
+      dockerfile: Dockerfile
+    container_name: synckeys
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/keys:/keys
+      - ./docker/data/sia:/sia-data
+    environment:
+      - SIAD_IP=10.10.10.10
```

Comment:
Can't you use `sia` alias that points to this ip?

Reply:
Yes, PJ just told me that's possible. I'm changing it. :)